custom_components/nordpool/__init__.py

```diff
@@ -135,24 +154,37 @@ async def new_day_cb(_):
 
             async_dispatcher_send(hass, EVENT_NEW_DAY)
 
-        async def new_hr(_):
-            """Callback to tell the sensors to update on a new hour."""
+        async def new_hr(tdo):
```

Comment:
I've spent couple minutes to understand what does "tdo" abbreviation mean and could not. Would be good to have a better name.

Reply:
My bet is on "time-and-date-object", which is a strange way to say "datetime".

Also that isn't used anywhere in the current code, it seems to be a leftover from somewhere.

Reply:
I added a commit that removes the unused tdo arguments from update_today, update_tomorrow and their callers.